lib/puma/cluster.rb

```diff
@@ -22,7 +23,8 @@ def initialize(launcher)
       @workers = []
       @next_check = Time.now
 
-      @phased_restart = false
+      @pending_phased_restart = false
```

Comment:
I renamed this because the code is extremely confusing right now and I think this helps clarify it some; there's the `in_phased_restart` local variable that _actually_ signifies a phased restart is happening, this is just a way of instructing the start of the main cluster loop to set that variable and begin the restart, it's immediately unset at that point so IMO it warrants the `pending` prefix.